helm/g2p-sandbox-fynarfin-SIT/values.yaml

```diff
@@ -227,6 +227,7 @@ ph-ee-g2psandbox:
         annotations:
           rollme: "{{ randAlphaNum 5 | quote }}"
       aws:
+        host: "http://minio:9000"
```

Comment:
remove these values from here, let it be configured only in the values.yaml of pg-ee-engine.
remove redudent code